js/mercado.js

```diff
@@ -350,10 +347,65 @@ module.exports = class mercado extends Exchange {
         };
     }
 
+    parseOHLCV (ohlcv, market = undefined, timeframe = '1m', since = undefined, limit = undefined) {
+        return [
+            ohlcv['timestamp'] * 1000,
+            parseFloat (ohlcv['open']),
+            parseFloat (ohlcv['high']),
+            parseFloat (ohlcv['low']),
+            parseFloat (ohlcv['close']),
+            parseFloat (ohlcv['volume']),
+        ];
+    }
+
+    async fetchOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let market = this.market (symbol);
+        let request = {
+            'precision': this.timeframes[timeframe],
+            'coin': market.id.toLowerCase (),
+        };
+        if (since !== undefined) {
+            request['from'] = parseInt (since / 1000);
+            if (limit !== undefined) {
+                request['to'] = (this.sum (request['from'], limit * this.parseTimeframe (timeframe)));
+            } else {
+                request['to'] = this.sum (this.seconds (), 1);
+            }
+        }
+        let response = await this.v4PublicGetCoinCandle (this.extend (request, params));
+        return this.parseOHLCVs (response.candles, market, timeframe, since, limit);
+    }
+
+    async fetchOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        if (symbol === undefined)
+            throw new ArgumentsRequired (this.id + ' fetchOrders () requires a symbol argument');
+        await this.loadMarkets ();
+        let market = this.market (symbol);
+        let response = undefined;
+        response = await this.privatePostListOrders (this.extend ({
+            'coin_pair': market['base'],
+        }, params));
+        let orders = response['response_data']['orders'];
+        for (let i = 0; i < orders.length; i++) {
+            orders[i] = this.parseOrder (orders[i]);
+        }
+        return orders;
+    }
+
+    async fetchTickers (symbols = undefined, params = {}) {
```

Comment:
We will have to remove this, because it isn't a "true" endpoint. If it requires more than one http request. In general we follow this rule: if it's a true endpoint – we can call it once, otherwise we set it to false and let the user fetch those tickers in a loop with `fetchTicker` in userland. Also, for that matter, we don't need the `.tickers` property in `describe()`, because we already have `.symbols` that are populated upon loading markets, as described here: 

- https://github.com/ccxt/ccxt/wiki/Manual#loading-markets
- https://github.com/ccxt/ccxt/wiki/Manual#symbols-and-market-ids

In some cases, we can emulate a call to one endpoint with a call to another endpoint, but we shouldn't really emulate one call with multiple calls. Because this breaks down the rate-limits, the async and parallel execution and all that.